test/functional/test_framework/util.py

```diff
@@ -497,7 +497,15 @@ def check_node_connections(*, node, num_in, num_out):
     assert_equal(info["connections_out"], num_out)
 
 def fill_mempool(test_framework, node, miniwallet):
-        """Fill mempool until eviction."""
+        """Fill mempool until eviction.
+
+        Allows for simpler testing of scenarios with floating minfee > minrelay
+        Requires -datacarriersize=100000 and
+        -maxmempool=5.
+        It will not ensure mempools become synced as it
+        is based on a single node and assumes -minrelaytxfee
+        is 1 sat/vbyte.
```

Comment:
Maybe assert for this at the beginning of the `fill_mempool`
```python
     assert_equal(node.getnetworkinfo()['relayfee'], Decimal('0.00001000'))
```

Reply:
done

Reply:
looking at this again, what do you think about just restarting the node with the `-datacarriersize=100000`, ` -maxmempool=5`, `-minrelaytxfee=0.00001000` after filling the mempool we then restart the node with default node settings?
Are there any advantage of delegating setting this to the caller?
If not then this will reduce duplication of all callers of `fill_mempool` settings this values.

Reply:
We can't restart the node after filling the memopool without resetting the floating minfee.

I could pull in the initial restart if we knew people didn't want any other extra_args.

Alternatively I could assert that datacarrier and maxmempool is set  in extra_args?

I think documentation is enough for now, leaving as is.